web/default/src/i18n/locales/en.json

```
@@ -2362,10 +2370,28 @@
     "Model not found": "Model not found",
     "Model performance metrics": "Model performance metrics",
     "Model Price": "Model Price",
-    "Model Price Not Configured": "Model Price Not Configured",
     "Model price is not configured. Please complete model pricing in settings.": "Model price is not configured. Please complete model pricing in settings.",
+    "Model Price Not Configured": "Model Price Not Configured",
     "Model prices": "Model prices",
     "Model prices reset successfully": "Model prices reset successfully",
+"Configure billing mode for image generation models. \"Per-token\" uses the standard token ratio. \"Per-resolution\" charges a flat price per image based on output size (1K ≤ 1024px, 2K ≤ 2048px, 4K > 2048px).": "Configure billing mode for image generation models. \"Per-token\" uses the standard token ratio. \"Per-resolution\" charges a flat price per image based on output size (1K ≤ 1024px, 2K ≤ 2048px, 4K > 2048px).",
+"Leave price fields empty to use the built-in default prices. Custom prices override the defaults.": "Leave price fields empty to use the built-in default prices. Custom prices override the defaults.",
+"Billing mode": "Billing mode",
+"Per-resolution": "Per-resolution",
```

Biome (2.4.15)

[error] The key Per-resolution was already declared.

(lint/suspicious/noDuplicateObjectKeys)